airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -87,9 +87,14 @@ def __init__(
         execution_role_arn: str | None = None,
         **kwargs,
     ):
-        if not exactly_one(job_flow_id is None, job_flow_name is None):
-            raise AirflowException("Exactly one of job_flow_id or job_flow_name must be specified.")
+        if (job_flow_id is None) and (job_flow_name is None):
+            raise AirflowException(
+                "Exactly job_flow_id or job_flow_name must be existed. "
```

Comment:
```suggestion
                 "At least one of job_flow_id or job_flow_name must be specified. "
```

If both are specified maybe we should log something to warn that something might not work as expected. (id takes precedence)